stable/kibana/templates/NOTES.txt

```diff
@@ -1,3 +1,18 @@
 To verify that {{ template "kibana.fullname" . }} has started, run:
 
-  kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "kibana.fullname" . }}"
+  kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "kibana.name" . }}"
+
+Kibana can be accessed:
+
```

Comment:
maybe add an if for when ingress is used as well?

Reply:
Makes sense. I am not actually sure what that would look like though in terms of commands to access it. Actually, I forgot in this redo that I should have added the load balancer service and removed the clusterip since this is an external facing dashboard. Do you agree?

Reply:
personally, I'd agree with that yes.